apps/fabric-website/src/pages/NotFoundPage/NotFoundPage.tsx

```diff
@@ -61,7 +61,7 @@ export class NotFoundPage extends React.Component<INotFoundPageProps, {}> {
                   ev => this._onInternalLinkClick(ev, '#/')
                 }
               >
-                UI Fabric Home
+                Fabric UI Home
```

Comment:
```suggestion
                Fluent UI Home
```

Reply:
might have missed this in fixing merge conflicts :D